### PR TITLE
update multiscale Sauvola k fixes from upstream

### PR DESCRIPTION
**Pull request by bertsky**
See (rather lengthy) discussion at https://gitlab.lrde.epita.fr/olena/olena/-/merge_requests/4

– this will have to be compensated for by callers in another PR to ocrd_olena.

### CODE REVIEW

**Comment by bertsky** [comment]
> this will have to be compensated for by callers in another PR to ocrd_olena.

@kba I'll do that PR as soon as this one is merged.

File: milena/Makefile.am [hunk 1]
```diff
@@ -40,7 +40,7 @@ tests:
 
 # Regen files recursively.
 include $(top_srcdir)/build-aux/regen-recursive.mk
-REGEN_SUBDIRS += tests
+REGEN_SUBDIRS += doc tests
```
Comment (kba) [inline milena/Makefile.am:43]: Was this intentional? Didn't we disable building the documentation?

**Review by kba — APPROVED** [review]
Looks good, feel free to merge. Should we start creating releases to keep track of these changes?

File: milena/Makefile.am (hunk 1 above)
Comment (bertsky) [reply milena/Makefile.am:43]: Yes, but this change is correct: First we disabled by patching the automake files in our clone, then Edwin made the respective permanent fix ee02495a3986a359b4a7e82b94e211b48c5ca196 upstream (by adding a configure switch `ENABLE_DOC` instead). We see the result of the merge here.

**Comment by bertsky** [comment]
> Should we start creating releases to keep track of these changes?

IMHO that's upstream's job (and I doubt they'd do it, since they want to supplant scribo with a new implementation).

And our submodule integration won't reference release tags/branches anyway, just commit checksums.

**Pull request merged**